src/openrct2/object/ObjectFactory.cpp

```diff
@@ -390,98 +394,81 @@ namespace ObjectFactory
         log_verbose("CreateObjectFromJsonFile(\"%s\")", path.c_str());
 
         Object* result = nullptr;
+        json_t jRoot;
         try
         {
-            auto jRoot = Json::ReadFromFile(path.c_str());
+            jRoot = Json::ReadFromFile(path.c_str());
```

Comment:
```suggestion
            json_t jRoot = Json::ReadFromFile(path.c_str());
```

Seems like you can reduce the scope of `jRoot` above